docker-compose.yml

```diff
@@ -18,8 +18,8 @@ services:
       SONAR_ES_BOOTSTRAP_CHECKS_DISABLE: 'true'
     volumes:
       - type: bind
-        source: ./sonar-ios-plugin/target/ecocode-sonar-ios-plugin-1.0.2-SNAPSHOT.jar
-        target: /opt/sonarqube/extensions/plugins/ecocode-sonar-ios-plugin-1.0.2-SNAPSHOT.jar
+        source: ./lib/ecocode-sonar-ios-plugin-1.1.0-SNAPSHOT.jar
```

Comment:
why change `target`to `lib` ? this modification implies that we must use `mvn package` (whihch produces jar file in `lib` directory) instead of `mvn compile` (whihch produces jar file in `target` directory) ... this change usage of tools scripts.

Reply:
I put it back to target :)